### ISSUE TITLE
How to set Parent Sessions officialy?

### ISSUE THREAD

**Issue by kazu0617** [Neos-Metaverse/NeosPublic#3101]
### Where have you searched for this information before making this issue?

neos-updates in NeosVR Discord, Wiki and whole of NeosVR Discord.

### Which part of the system do you need information on?

Headlessのセッションを作成する際、parentsessionIdsを設定しても人によって挙動が違います
（自分の場合はnestedsessionidsに入るにもかかわらずNeos内から確認すると誰もから見えるようになっていました）。
そのため、どう設定するのが公式の答えなのか教えて欲しいです。

When I create a Headless session, even if I set the parentsessionIds, it behaves differently depending on the person.
(In my case, it was visible to everyone when I checked from within Neos even though it was in nestedsessionids).
So I'd like to know what the official answer is.

### What information do you need to know?

The NeosVR Wiki says that if you set the `parentsessionIds` and are friends with the session you are nesting from, then you are good to go, but in reality, that is not enough to make it work correctly.

So I'd like to know what the official answer is.
for example(not work properly / friends with `U-kiyomaruHDL`):
```
    {
      "accessLevel": "Anyone",
      "loadWorldURL": "neosrec:///G-Neos-East-Japan/R-95d04d7b-322f-47ea-a18e-0fc1f473d41e",
      "idleRestartInterval": 1800.0,
      "awayKickMinutes": 3.0,
      "parentSessionIds": [
        "S-U-kiyomaruHDL:JPTutorial"
      ]
    }
```

### Do you plan to format and contribute the provided information to Wiki?

Yes

**Comment by shadowpanther** [comment]
In the nested session, you have to setup `parentSessionIds` like you are doing.
The headless accounts of the parent session and the nested session have to be mutual friends.
To request a friend, you have to log in with one of your headless accounts on a full client, the headless can only accept friends, not request them.

Source: setting up sessions for the Metaverse Meetup.

**Comment by kazu0617** [comment]
> 
> 
> In the nested session, you have to setup `parentSessionIds` like you are doing. The headless accounts of the parent session and the nested session have to be mutual friends. To request a friend, you have to log in with one of your headless accounts on a full client, the headless can only accept friends, not request them.
> 
> Source: setting up sessions for the Metaverse Meetup.

That's right, but I've already set up those three things. However, I just checked and it seems that Headless and friends may be displayed regardless of the parentSessionIds setting.

**Comment by kazu0617** [comment]
この問題ですが、最終的に"parentSessionIds"が指定されているセッションの場合、ネスト先のセッションがなかったとしても見た目上表示されないようにしてもらうTweakを出すのがいい気がしてきました（場合によっては表示され、場合によっては表示されないので。画像の例の場合、本来であれば一番上以外はすべて見えないはずなのですが、すべて見えてしまっています。この見える問題は時間を置くと改善される場合もあり、また突然出てくることもあります）。

As for this problem, I think it would be a good idea to issue a tweak to make it so that sessions with "parentSessionIds" will not be visible even if there are no nested sessions (in some cases they will be visible, in some cases they will not). In the case of the example in the image. In the case of the example in the image, everything except the top one should be invisible, but it is. (This visibility problem may improve with time, or it may come up suddenly.
![image](https://user-images.githubusercontent.com/6259214/135322594-b589bf3a-c4c5-4cf8-a9e1-685420e7d9a1.png)
